src/main/java/com/laborete/LaboreteAPI/profile/models/UserAvatarDTO.java

```diff
@@ -15,4 +15,6 @@ public class UserAvatarDTO {
     private String name;
     @JsonProperty("size")
     private Long size;
+    @JsonProperty("extension")
```

Comment:
`UserAvatarDTO` and `UserBackgroundDTO` are no longer used. This is called _dead code_. Remove them. If we need them in the future, we will use git to restore them.